lib/js/components/Well/Well.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div
+		class="well"
+		:class="{
+			'-medium': WELL_SIZES.MEDIUM === size,
+			'-small': WELL_SIZES.SMALL === size,
+		}"
+	>
+		<div class="well__content">
+			<slot />
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/colors/tokens';
+
+.well {
+	background-color: $color-neutral-background;
+	border-radius: $radius-m;
+
+	&.-medium {
+		padding: $space-s;
+	}
+
+	&.-small {
+		padding: $space-xs;
+	}
+
+	&__content {
+		padding: $space-xxs $space-xxxxs;
+	}
+}
+</style>
+
+<script>
+import { WELL_SIZES } from './Well.consts';
+
+export default {
+	name: 'Well',
+	props: {
+		size: {
```

Comment:
in Figma it's called `padding` and has `none/s/m` options

Reply:
https://github.com/bethinkpl/design-system/pull/284/commits/960901a37711277a089fd4add7f19b67df5019c2